eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/serialization/temp/parcelable/Everything.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2017 SchedJoules
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.schedjoules.eventdiscovery.framework.serialization.temp.parcelable;
+
+import android.os.Parcel;
+
+import com.schedjoules.eventdiscovery.discovery.EventDiscovery;
+
+
+/**
+ * @author Gabor Keszthelyi
+ */
+class Everything implements EventFilter
```

Comment:
Updated, although this is just temporary code for that here, I will remove them.